### DISCUSSION TITLE
Render newlines in the message in the web app

### DISCUSSION THREAD

**Discussion by colinangusmackay** [datalust/seq-tickets#1079] [Ideas]
Some log messages come with newlines in the message. However, when this is rendered to the browser in Seq the newlines are not rendered so the message starts to look confusing.

e.g.
Using `Microsoft.Logging.Extensions`, here's the console output of a log message:
```
    info: Example.Startup[0]
      The following connection strings were found: MainDatabase, PostCodeLookupDatabase.
      Connection string (named MainDatabase) parameters:
       * [MainDatabase].[server] = dev.my-database-server.my-company.com
       * [MainDatabase].[database] = myDataBase
       * [MainDatabase].[user id] = myUsername
       * [MainDatabase].[password] = **********

      Connection string (named PostCodeLookupDatabase) parameters:
       * [PostCodeLookupDatabase].[provider] = MSOLEDBSQL
       * [PostCodeLookupDatabase].[server] = tcp:AvailabilityGroupListenerDnsName,1433
       * [PostCodeLookupDatabase].[multisubnetfailover] = Yes
       * [PostCodeLookupDatabase].[applicationintent] = ReadOnly
       * [PostCodeLookupDatabase].[database] = MyDB
       * [PostCodeLookupDatabase].[integrated security] = SSPI
       * [PostCodeLookupDatabase].[connect timeout] = 30
```

And here is the same output in Seq:
![image](https://user-images.githubusercontent.com/1372771/104847356-6de6fe80-58d7-11eb-9e62-db1a58cfebc1.png)

**Comment by nblumhardt** [comment]
Thanks for the suggestion; it's currently "by design" but could it use another look 👍

**Comment by colinangusmackay** [comment]
Perhaps, if some folks don't want it, it could be a configuration toggle. Or when the log entry is expanded it gets re-rendered with the newlines.
